docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+Before starting work on some new feature/bug fix please create an Issue in our GitHub repository and describe, what you
+are planning or what the issue is. There we can discuss if what you are proposing is something we would like to add and
+what the best way to do that is.
+
+## Review process
+
+- File a Pull Request (PR) with a number of well-defined clearly described commits (following what is described
+  in [this article](https://chris.beams.io/posts/git-commit/) is a good start). Multiple commits per PR are allowed, but
+  please do not include revert commits, etc. Use rebase.
+- Make sure that the code is well formatted - use `iSort` and `Black` for that:
+
+```
+pipenv shell
+isort src
+black src
```

Comment:
Hmm I don't use black like this. This is for VSCode? Maybe don't add this helpful commands in case that it might be confusing for someone else?

Reply:
https://github.com/StreetOfCode/discord-bot/pull/20/commits/4a7fce8685365c920eb8076ca24b7d7cda6fe138